Clockwise/CommandHandler.cs

```diff
@@ -70,22 +70,22 @@ public static class CommandHandler
                               async d => await handler.Handle(d)));
     }
 
-    public delegate  Task<ICommandDeliveryResult> HandlerDelegate<in T>(ICommandDelivery<T> delivery);
+    public delegate Task<ICommandDeliveryResult> CommandHandler<in T> (ICommandDelivery<T> delivery);
```

Comment:
I think this name is confusing because we already have `ICommandHandler<T>`, which is an interface. By the name, I would expect this to be a class implementing that interface.

Maybe something like `HandleCommand<T>`?